app/email.py

```diff
@@ -1,21 +1,16 @@
-from threading import Thread
-
-from flask import current_app, render_template
+import os
+from flask import render_template
 from flask.ext.mail import Message
 from . import mail
+from app import create_app
 
 
-def send_async_email(app, msg):
+def send_email(recipient, subject, template, **kwargs):
+    app = create_app(os.getenv('FLASK_CONFIG') or 'default')
```

Comment:
why create app? Can't you grab the current app context with `current_app._get_current_object()` Per the Flask docs it refs the application object directly and grabs the current state and variable scope of the app as well.